pkg/sdk/testint/grants_integration_test.go

```diff
@@ -511,8 +511,6 @@ func TestInt_GrantAndRevokePrivilegesToDatabaseRole(t *testing.T) {
 		})
 		require.NoError(t, err)
 		// Expecting two grants because database role has usage on database by default
-		require.Equal(t, 2, len(returnedGrants))
```

Comment:
let's not remove the check, but check if it's at least 2

Also, if you do something like this please remove the comment too.

Reply:
Fixed.